src/Rewriter/Util/plugins/definition_by_tactic.ml.v813

```diff
@@ -5,7 +5,8 @@ let make_definition_by_tactic sigma ~poly (name : Names.Id.t) (ty : EConstr.t) (
   let info = Declare.Info.make ~poly () in
   let lemma = Declare.Proof.start ~cinfo ~info sigma in
   let lemma, _ = Declare.Proof.by tac lemma in
-  let ids = Declare.Proof.save ~proof:lemma ~opaque:Vernacexpr.Transparent ~idopt:None in
+  let pm = Declare.OblState.empty in
+  let _pm, ids = Declare.Proof.save ~pm ~proof:lemma ~opaque:Vernacexpr.Transparent ~idopt:None in
```

Comment:
We have save_regular now btw

Reply:
I was lazy but in the end, I updated it.